bitnami/contour/Chart.yaml

```
@@ -25,4 +25,4 @@ maintainers:
 name: contour
 sources:
   - https://github.com/bitnami/charts/tree/main/bitnami/contour
-version: 12.1.1
+version: 12.1.2
```

As you are adding new parameters to the chart

```suggestion
version: 12.2.0
```